src/style.css

```diff
@@ -1,3 +1,78 @@
+:root {
+  --blue: #00acdc;
+  --black: #000;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header__logo {
+  height: 60px;
+}
+
+.header__img {
+  margin: 10px 0;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+}
```

Comment:
redundant
```suggestion
```

Reply:
Still not fixed
<img width="841" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/584545d2-6d94-4f62-aa66-d5d39697875d">